docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+  before pushing to GitHub.
+- A Python interpreter of 3.7 or greater for testing wheels locally.
+
+Before getting started it's helpful to have a working understanding of the following:
+- ITK external modules
+  (see [README discussion](../README.md#motivation));
+- [GitHub Actions](https://docs.github.com/en/actions) and
+  [continuous integration](https://docs.github.com/en/actions/automating-builds-and-tests/about-continuous-integration)
+  in general;
+- [GitHub Actions Reusable Workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows).
+
+The following tools and concepts are used in `ITKRemoteModuleBuildTestPackageAction`
+and may be helpful or required for contributing:
+
+- [Bash](https://www.gnu.org/software/bash/) and
+  [PowerShell](https://learn.microsoft.com/en-us/powershell/) scripting;
+- [CMake](https://cmake.org/) and [Ninja](https://ninja-build.org/) build tools;
+- [CTest](https://cmake.org/cmake/help/latest/manual/ctest.1.html) and 
+  [CDash](https://cmake.org/cmake/help/book/mastering-cmake/chapter/CDash.html)
+  testing and reporting tools;
+- [Python](https://www.python.org/) scripting;
+- [Python wheel](https://wheel.readthedocs.io/en/stable/) packaging;
+
+## Creating or Modifying a Workflow
+
+GitHub Actions requires that workflows are placed in the [`.github/workflows`](../.github/workflows) directory.
+This is where you will find existing workflows and possibly add new workflows.
+
+To contribute to `ITKRemoteModuleBuildTestPackageAction` you will need to
+first [fork](https://docs.github.com/en/github-ae@latest/get-started/quickstart/fork-a-repo)
+the repository to your user account on GitHub and then
+[clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+the fork to your local machine.
+
+### Creating a Workflow
+
+`ITKRemoteModuleBuildTestPackageAction` is a central repository for Github Actions
+reusable workflows for ITK external modules. Before creating a workflow, ask yourself
+the following questions:
+1. Does my workflow address a problem for ITK external modules that is not
+   already solved?
+2. Does my workflow provide functionality that will generalize to various ITK external modules?
+3. Does my workflow provide functionality related to continuous integration such as
+   building, packaging, and/or testing steps?
+
+If the answer to all of the above is "yes", then continue on with your new workflow!
+
+Reference
+[Github Actions Reusable Workflow documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow)
+to get started. You will need to:
+1. Create a workflow in the `.github/workflows` folder;
+2. Define workflow inputs;
+3. Define how jobs will be launched;
+4. Define steps that a given job will take.
+
+### Modifying a Workflow
+
+Workflows will need to be maintained and updated over time as tools
+and technology progress. The best approach to modifying a workflow
```

Comment:
If a change may be integrated into an existing workflow so that other ITK external modules can benefit from beyond the provided functionality, then changes to existing workflows must be privileged. Similarly, workflows (...)"

Reply:
Could you please clarify what you mean by this suggestion?

Reply:
I meant that it might be more clear to start as:

> If a change may be integrated into an existing workflow so that other ITK external modules can benefit from beyond the provided functionality, then changes to existing workflows must be privileged. Similarly, workflows will need to (...)

Reply:
Thanks. If I'm understanding correctly, in my opinion discussions around privileges are already covered in 1) workflow motivation ("Does my workflow provide functionality that will generalize" and 2) PR  setup ("After a repository maintainer reviews, approves, and merges...". The "modifying a workflow" section by itself should focus on the editing process.

Reply:
Ah, I understand now, thanks for clarifying. I agree with your assessment and see where discussion around motives was missing from the "Modifying" section. I've added a couple of notes on what types of modifications will be accepted, please take a look and let me know what you think.

In particular I agree that we want modifications to be "privileged" in the sense that 1) maintainer approval is required and 2) modifications must meet certain criteria in support of the greater ITK external module ecosystem. We want to discourage individual contributors from adding targeted changes that will only benefit their own external modules.

Reply:
Went through the new wording: looks reasonable. Thanks Tom. Resolving.